tests/jquery.flot.hover.Test.js

```diff
@@ -258,5 +258,81 @@ describe("flot hover plugin", function () {
 
             expect(getEntireCanvasData(canvas)).toContainPixelColor(rgba(10, 20, 30, 1));
         });
+
+        it('should call hooked function when canvas is hovered over', function() {
```

Comment:
Should this test be in jquery.flot.Test.js?

Reply:
I don't believe so, as the hook is going to be called by the hover plugin's calling into flot's `findNearbyItems`.